web/modules/custom/dpl_admin/src/Controller/AdminPages.php

```diff
@@ -0,0 +1,102 @@
+<?php
+
+namespace Drupal\dpl_admin\Controller;
+
+use Drupal\Core\Controller\ControllerBase;
+use Drupal\Core\Entity\EntityTypeManagerInterface;
+use Drupal\Core\Url;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Custom admin pages.
+ */
+class AdminPages extends ControllerBase {
+
+  /**
+   * The entity type interface.
+   *
+   * @var \Drupal\Core\Entity\EntityTypeManagerInterface
+   */
+  protected $entityTypeManager;
+
+  /**
+   * {@inheritdoc}
+   */
+  public function __construct(EntityTypeManagerInterface $entity_type_manager) {
+    $this->entityTypeManager = $entity_type_manager;
+  }
```

Comment:
Consider changing this to use constructor property promotion.

Reply:
I tried, but I couldnt get it to work.
Even if I could, i dont see the big advantage